src/main/java/th/ac/kmitl/it/soa/group3/model/supplychaintradetransaction/PostalTradeAddressModel.java

```diff
@@ -0,0 +1,120 @@
+package th.ac.kmitl.it.soa.group3.model.supplychaintradetransaction;
+
+public class PostalTradeAddressModel {
+
+    public String postCode;
+    public String buildingName;
+    public String addressLine1;
```

Comment:
ยังงงกับ addressLine1, addressLine2 อยู่ดี ว่าจะใส่อะไร เพราะเหลือแค่บ้านเลขที่กับชื่อตึก

Reply:
ในเอกสารมันบอกงี้อะครับ แล้วจะให้ผมเขียนอะไรอะ ที่อยู่บรรทัด1-2
![0f66a8c98dfb25c76315f5b2419c3966.png](https://www.img.in.th/images/0f66a8c98dfb25c76315f5b2419c3966.png)

Reply:
ไม่เคลียร์มา ว่า addressLine1 กับ addressLine2 ต่างกันยังไง